python/taichi/lang/kernel_arguments.py

```diff
@@ -40,6 +41,14 @@ def decl_scalar_arg(dtype):
     return Expr(_ti_core.make_arg_load_expr(arg_id, dtype))
 
 
+def decl_matrix_arg(matrixtype):
+    tmp = Matrix(n=matrixtype.n, m=matrixtype.m)
+    for i in range(matrixtype.n):
+        for j in range(matrixtype.m):
+            tmp.set_entry(i, j, decl_scalar_arg(matrixtype.dtype))
+    return tmp
```

Comment:
This way of constructing `Matrix` will be deprecated soon :-) 
```suggestion
    return Matrix([[decl_scalar_arg(matrixtype.dtype) for _ in matrixtype.m] for _ in matrixtype.n])
```